pubspec.lock

```diff
@@ -88,13 +109,27 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "1.8.0+1"
+  provider:
```

Comment:
where do you use this package/what's it for?

Reply:
comes as a dependency from flutter_bloc